.vsts-ci/linux.yml

```
@@ -30,6 +30,7 @@ pr:
     - .vsts-ci/windows.yml
     - .vsts-ci/windows/*
     - test/common/markdown/*
+    - tools/perf/*
```

This should be test/perf/*
```suggestion
    - test/perf/*
```